.github/workflows/deploy_knockdog.yml

```
@@ -0,0 +1,97 @@
+
+      - name: Create empty .env.production
+        run: |
+          mkdir -p apps/knockdog
+          echo "PUBLIC_ASSET_PREFIX=${{ secrets.PUBLIC_ASSET_PREFIX }}" > apps/knockdog/.env.production
+
+      - name: Build Knockdog
+        run: pnpm dlx turbo build --filter=knockdog...
+
+      # AWS 자격증명 설정 (S3 업로드용)
+      - name: Configure AWS credentials for S3
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)